### PR TITLE
Update text for purchase modal

### PR DESCRIPTION
**Pull request by nozomuplus**
# Changes
- Update English text for the modal before purchase
- Refactor displaying alert

# TODO (maybe after the first release)
Lock & unlock related process should be segregated into View and Presenter properly. 

purchase_modal

### CODE REVIEW

File: ZIGSIMPlus/Views/CommandSelectionViewController.swift [hunk 1]
```diff
@@ -171,40 +166,45 @@ final class CommandSelectionViewController: UIViewController {
...
-        alert.setValue(aText, forKey: "attributedMessage")
+        return NSMutableAttributedString(attributedString: markdownParser.parse(msg))
     }
     
-    private func isAvailablePremiumCommands() -> Bool {
-        if unAvailablePremiumCommands.count != 0 ||
-            !VideoCaptureService.shared.isDepthRearCameraAvailable() ||
-            !VideoCaptureService.shared.isDepthFrontCameraAvailable() {
-            return false
+    private var unavailableFunctionCount: Int {
```
Comment (YoneyamaShunpei) [inline ZIGSIMPlus/Views/CommandSelectionViewController.swift]: ここではPremiumなコマンドのみで使用できるかを判断しているので`unAvailablePremiumCommands`とした方がもっと分かりやすくなると思います。

Comment (nozomuplus) [reply ZIGSIMPlus/Views/CommandSelectionViewController.swift]: Thanks! Updated in d4205f7 .
I think this is called as "function", not "command" (I think you wrote that).

Comment (YoneyamaShunpei) [reply ZIGSIMPlus/Views/CommandSelectionViewController.swift]: 修正いただきありがとうございます。
ここはfunctionであるべきですね。

File: ZIGSIMPlus/Views/CommandSelectionViewController.swift [hunk 2]
```diff
@@ -171,40 +166,45 @@ final class CommandSelectionViewController: UIViewController {
...
-    private func setAlertMessage(_ message: String, _ alertType: alertModalType) {
-        var msg = message
-        if  alertType == alertModalType.premium && !isAvailablePremiumCommands() {
-            msg = premiumTextBody + "\n\nBut the following function can not be used on your device."
+    private func getAlertMessageForPurchase() -> String {
+        var message = premiumTextBody
+        if  unavailablePremiumFunctionCount > 0 {
+            message += (unavailablePremiumFunctionCount >= 2
+                ? "\n\nThe following functions don't work on this device:"
+                : "\n\nThe following function doesn't work on this device:")
```
Comment (fand) [inline ZIGSIMPlus/Views/CommandSelectionViewController.swift]: I think `The following` should be `Following` ...? 🤔

Comment (nozomuplus) [reply ZIGSIMPlus/Views/CommandSelectionViewController.swift]: Adding "The" seems to be correct.
https://ell.stackexchange.com/questions/100202/the-following-vs-following

Comment (fand) [reply ZIGSIMPlus/Views/CommandSelectionViewController.swift]: 😳

**Comment by nozomuplus** [comment]
Thanks for your reviews!

**Pull request merged**